### DISCUSSION TITLE
[Esame 23 Febbraio 2024] Esercizio 3

### DISCUSSION THREAD

**Discussion by Elia-Belli** [sapienzastudentsnetwork/calcolo-delle-probabilita#271] [Esami - Bertini @ Informatica]
**Esercizio 3.** Ogni giorno arriva un numero di richieste dato da una variabile di poisson di parametro 5, ogni richiesta ha probabilità ⅔ di essere di tipo A e probabilità ⅓ di essere di tipo B.

**(i)** Calcolare la probabilità che in un giorno arrivino esattamente 5 richieste di tipo A.

**(ii)** Calcolare il valore atteso delle richieste di tipo A e B.

**(iii)** Sapendo che sono arrivate 9 richieste, qual'è la probabilità che 6 di queste siano di tipo A?

**(iv)** Sapendo che sono arrivate almeno 3 richieste, qual'è la probabilità che le prime 3 siano di tipo A?

**Comment by Traster-uni** [comment]
![IMG_20240304_164421](https://github.com/sapienzastudentsnetwork/calcprob2324/assets/84441861/f8e83548-d23d-4533-af47-d751746a2941)
Per favore correggete, che io non sono affatto sicuro di questa soluzione nonostante mi sia basato su ciò che è stato detto in classe (appunti) e il libro (Ross)

**Comment by MarcoLinardi** [comment]
![IMG_0344](https://github.com/user-attachments/assets/e1ddcdd5-6648-468a-8a4d-4d0218f684b7)

**Comment by CiottoloMaggico** [comment]
![immagine](https://github.com/user-attachments/assets/4aee157f-ac2c-4f97-be01-c2f746e4ce84)
![immagine](https://github.com/user-attachments/assets/ddf5a765-6180-4a14-a0c4-d188718d72e1)
